docs/narr/project.rst

```diff
@@ -856,12 +860,19 @@ also informs Python that the directory which contains it is a *package*.
    WSGI application.  This function is meant to be called by the
    :term:`PasteDeploy` framework as a result of running ``pserve``.
 
-   Within this function, application configuration is performed.
+   :term:`PasteDeploy` configures logging from the ``.ini`` file
+   before ``main`` is called.  Within ``main`` further application
+   configuration is performed.
```

Comment:
This flow is awkward. We talk about `main`, then jump to PasteDeploy `ini`, then back to `main`.

Please take a look at lines 658-666. I think that it would be better to insert a subsection heading up there, like so:

```
.. index:: 
    single: ini file processing
    pair: extensions; configuration
    pair: logging; configuration
    pair: PasteDeploy; configuration

.. _myproject_ini_logging:

Logging configuration
^^^^^^^^^^^^^^^^^^^^^

The sections after ``# logging configuration`` represent Python's standard
```

Then refer to that, like so:

```
Within this function, application configuration is performed, except for :ref:`logging as described previously <myproject_ini_logging>`.
```

p.s.: Narrative docs do not need to be hard-wrapped to 79 columns. It is [preferred not to wrap sentences, and to put each sentence on its own line](https://docs.pylonsproject.org/projects/docs-style-guide/#use-of-whitespace). This makes diffing narrative docs much easier because whitespace noise from wrapping is eliminated.